### PR TITLE
[terra-framework] Removed inline style for dev-site components Part 5

### LINKED ISSUE #657: Remove inline styles in dev site components
# Feature Request

## Description

We want to remove inline styles from our dev site components to set a better example for consumers of terra.  As a replacement, we should pass in classes and use css to style the components.

### PR DESCRIPTION
**Pull request by pranav300**
### Summary
Removed all inline styles and replaced the same styles with external CSS for the following packages:

1. Application Header Layout
2. Application Menu Layout
3. Application Name
4. Navigation Side Menu

Partially resolves #657

### CODE REVIEW

File: packages/terra-navigation-side-menu/src/terra-dev-site/doc/example/NavigationSideMenuExample.scss [hunk 1]
```diff
@@ -0,0 +1,15 @@
+:local {
+  .content {
+    background-color: #ffc0cb;
+    height: 100%;
+    position: relative;
+    width: 100%;
+  }
```
Comment (ShettyAkarsh) [inline packages/terra-navigation-side-menu/src/terra-dev-site/doc/example/NavigationSideMenuExample.scss]: We can import this from NavigationSideMenuDocCommon

Comment (pranav300) [reply packages/terra-navigation-side-menu/src/terra-dev-site/doc/example/NavigationSideMenuExample.scss]: fixed here https://github.com/cerner/terra-framework/pull/693/commits/fe6b8b31fa9c34b77574262f68e1b3ed346dff49.

File: packages/terra-application-name/src/terra-dev-site/doc/example/ApplicationHeaderNameStandard.jsx [hunk 2]
```diff
@@ -24,6 +24,7 @@ const ApplicationHeaderNameStandard = () => (
     extensions={<Placeholder text="Extensions" type="header-extensions" />}
     navigation={<Placeholder text="Content" type="default" />}
     utilities={<Placeholder text="Utiltities" type="utiltities" />}
+
```
Comment (emilyrohrbough) [inline packages/terra-application-name/src/terra-dev-site/doc/example/ApplicationHeaderNameStandard.jsx]: nit: remove extra line

Comment (pranav300) [reply packages/terra-application-name/src/terra-dev-site/doc/example/ApplicationHeaderNameStandard.jsx]: Fixed here https://github.com/cerner/terra-framework/pull/693/commits/f4e8b011368aedefbf07de34a063e55a24cb196d.

**Pull request merged**